src/widgets/settingspages/ModerationPage.cpp

```diff
@@ -69,8 +70,23 @@ ModerationPage::ModerationPage()
 
     auto logs = tabs.appendTab(new QVBoxLayout, "Logs");
     {
-        logs.append(this->createCheckBox("Enable logging",
-                                         getSettings()->enableLogging));
+        QCheckBox *enableLogging = this->createCheckBox(
+            "Enable logging", getSettings()->enableLogging);
+        logs.append(enableLogging);
+        QCheckBox *customLogging =
+            this->createCheckBox("Enable logging for specific channels",
```

Comment:
Phrasing of the setting should be more along the lines of `Only log channels listed below`, and it should be moved to be just above the list of channels